Cargo.toml

```diff
@@ -6,10 +6,19 @@ edition = "2021"
 [[bin]]
 name = "grpc-as"
 path = "bin/grpc-as/src/main.rs"
+required-features = [ "rvps-server", "rvps-proxy", "tokio/rt-multi-thread" ]
+
+[[bin]]
+name = "rvps"
+path = "bin/rvps/src/main.rs"
+required-features = [ "rvps-server", "rvps-proxy", "tokio/rt-multi-thread" ]
 
 [features]
-default = [ "in-toto" ]
-in-toto = []
+default = [ "rvps-server" ]
+rvps-server = [ "in-toto" ]
```

Comment:
Why not directly use `"dep:in-toto` here, so that `in-toto` feature can be remolved below.

Reply:
My initial idea is to make `in-toto` as a option feature. Here I made a mistake, let me fix into
```
default = [ "rvps-server" ]
rvps-server = []
rvps-proxy = [ "tonic" ]

in-toto = [ "dep:in-toto" ]
```

In this way, we can choose whether to enable `in-toto` support